Appyo-Poste/Poste
# Us189 Change Password

To test:
Login -> Click gear icon on dashboard -> Click Account Settings -> enter old password, new password and confirm new password. 

Backend server URL implemention on branch: 
https://github.com/Appyo-Poste/PosteBackend/tree/us189_changePassword_fmehdad
